.github/workflows/ui_smoke_tests.yml

```diff
@@ -43,9 +44,10 @@ jobs:
           name: UI Smoke Test Report
           path: reports/report.xml
           reporter: java-junit
-      - name: Action Summary
+      - name: Summary
         if: always()
         run: |
           type summary.md | Out-File -FilePath $env:GITHUB_STEP_SUMMARY -Append
-          echo "" >> $GITHUB_STEP_SUMMARY
-          echo ":package: Trigerred by: [${{ github.event.workflow_run.name }}](${{ github.event.workflow_run.html_url }})" >> $env:GITHUB_STEP_SUMMARY
+          echo "Actor: @${{ github.actor }}" >> $env:GITHUB_STEP_SUMMARY
+          echo "Workflow Run: [${{ github.event.workflow_run.name }}](${{ github.event.workflow_run.html_url }})" >> $env:GITHUB_STEP_SUMMARY
```

Comment:
so are any of these github event data user controllable?
https://securitylab.github.com/research/github-actions-untrusted-input/